media-libs/libmygpo-qt/libmygpo_qt-1.1.0.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="Qt/C++ library wrapping the gpodder.net webservice"
+DESCRIPTION="libmygpo-qt is a Qt/C++ Library that wraps the gpodder.net Web API \
+http://wiki.gpodder.org/wiki/Web_Services/API_2"
+HOMEPAGE="http://wiki.gpodder.org/wiki/Libmygpo-qt"
+COPYRIGHT="2010-2018 Stefan Derkits, Christian Wagner & Felix Winter"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/gpodder/libmygpo-qt/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="2eab976efbbdd39bae1b436e7dcd2ddffd35241432094f29b69308840a938459"
+SOURCE_DIR="libmygpo-qt-$portVersion"
+
+ARCHITECTURES="?all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libmygpo_qt$secondaryArchSuffix = $portVersion
+	lib:libmygpo_qt5$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libmygpo_qt${secondaryArchSuffix}_devel = $portVersion
+	devel:libmygpo_qt5$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	libmygpo_qt$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Core$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:dot
+	cmd:doxygen
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	cmake -B build -S . -DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DINCLUDE_INSTALL_DIR=$includeDir/mygpo-qt5 \
+		-DLIB_SUFFIX="/${secondaryArchSuffix/_/}" \
```

Comment:
```suggestion
		-DLIB_SUFFIX="$secondaryArchSubDir" \
```

Reply:
Wasn't sure there, I guess a few others should be "fixed" for this too when time is there?